angelsindustries/prototypes/overrides/components-bobs-entity-update.lua

```diff
@@ -28,177 +28,177 @@ if angelsmods.industries.components then
             name = "yellow-filter-inserter", -- filter inserter
             ingredients = {
               {"!!"},
-              {"inserter", 1},
-              {"circuit-grey", 2},
-              {"tin-plate", 2},
+              {type = "item", name = "inserter", amount = 1},
+              {type = "item", name = "circuit-grey", amount = 2},
+              data.raw["tin-plate"] and {type = "item", name = "tin-plate", amount = 2} or nil
             }
           },
           {
             name = "long-handed-inserter", -- fast inserter
             ingredients = {
               {"!!"},
-              {"inserter", 1},
-              {"circuit-red-loaded", 1},
-              {"angels-servo-motor-1", 2},
-              {"bronze-alloy", 2},
+              {type = "item", name = "inserter", amount = 1},
+              {type = "item", name = "circuit-red-loaded", amount = 1},
+              {type = "item", name = "angels-servo-motor-1", amount = 2},
+              data.raw["bronze-alloy"] and {type = "item", name = "bronze-alloy", amount = 2} or nil
```

Comment:
instead of checking if bronze alloy exists, you should check the smelting trigger `angelsmods.trigger.smelting_products["bronze"].plate`